app/controllers/manual_sections_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ManualSectionsController < ApplicationController
+  def show
+    @section = Section.find(content_id: params[:content_id])
```

Comment:
As this action is routed via a nested url (`/manuals/:manual_content_id/sections/:content_id`) do we want to make sure that the supplied `content_id` is really a child of the manual with the supplied `manual_content_id`?  Probably something like `raise <whatever-our-404-exception-is> if @section.manual_content_id != params[:manual_content_id]`